README.md

```diff
@@ -126,6 +127,49 @@ client = Redis.new(semian: {
 })
 ```
 
+#### Net::HTTP
+For the `Net::HTTP` specific Semian adapter, since many external libraries may create
+HTTP connections on the user's behalf, the parameters are instead provided
+by calling specific functions in `Semian::NetHTTP`.
+
+To give Semian parameters, assign a `proc` to `Semian::NetHTTP.raw_semian_options`
+that takes a `semian_identifier` like `http_127_0_0_1_80` or `http_github_com_80`,
+and returns a `Hash` with keys as follows, perhaps in an initialization file.
+
+```ruby
+SEMIAN_PARAMETERS = { tickets: 1,
+                      success_threshold: 1,
+                      error_threshold: 3,
+                      error_timeout: 10 }
+Semian::NetHTTP.raw_semian_options = proc do |semian_identifier|
+  # Let's make it only active for github.com
+  if semian_identifier == "http_github_com_80"
```

Comment:
@kyewei the reason to get this in master is for @sunblaze's PR, we're not getting it in Shopify before BFCM.

As for @byroot's point, I think it's good—you may want to make decisions about grouping a larger set of domains in the same Semian resource, and you cannot do that with the current design. Invoking a closure on every single resource call that returns the resource name and the config, and make that mandatory, seems like the way to go.